integration-tests/tests/run-catalog-workflow.js

```diff
@@ -21,6 +21,5 @@ const testCatalogFlowFn = _.flow(
 registerTest({
   name: 'run-catalog',
   fn: testCatalogFlowFn,
-  timeout: 2 * 60 * 1000,
-  targetEnvironments: []
+  timeout: 2 * 60 * 1000
```

Comment:
Since we're re-enabling these two tests, have we run them with `test-flakes`

Reply:
I have not. Can do. I don't think they've been flaky though?

Reply:
We wouldn't know given the tests haven't being run?

Reply:
They were running for a good long while. But yeah, the flow is different now. I can absolutely run a test.